rules/filename-case.js

```diff
@@ -144,7 +142,15 @@ function englishishJoinWords(words) {
 }
 
 const create = context => {
-	const chosenCases = getChosenCases(context);
+	const options = context.options[0] || {};
+	const chosenCases = getChosenCases(options);
+	const ignore = (options.ignore || []).map(item => {
+		if (item instanceof RegExp) {
+			return item;
+		}
+
+		return new RegExp(item, 'u');
```

Comment:
Shouldn't we escape the string? otherwise

```js
new RegExp('[fooBar].js').test('f.js')
// -> true
```

Reply:
Huh? The string support is meant to allow regex in a string, for use with JSON config, where you cannot use regex literal.

Reply:
right, forgot